tests/sps/validation/test_dates.py

```diff
@@ -461,30 +237,392 @@ def test_is_sorted_a_incomplete_date_list(self):
+                                <date date-type="rev-request">
+                                    <day>04</day>
+                                    <month>01</month>
+                                    <year>1998</year>
+                                </date>
+                                <date date-type="rev-recd">
+                                    <day>24</day>
+                                    <month>05</month>
+                                    <year>1998</year>
+                                </date>
+                                <date date-type="accepted">
+                                    <day>06</day>
+                                    <month>06</month>
+                                    <year>1998</year>
+                                </date>
+                                <date date-type="corrected">
+                                    <day>01</day>
+                                    <month>06</month>
+                                    <year>2012</year>
+                                </date>
+                            </history>
+                        </article-meta>
+                    </front>
+                </article>
+                """
+        )
         expected = [
-            {'input': {'year': '1998', 'month': '01', 'type': 'received'},
-             'result': 'error',
-             'element': 'day',
-             'message': "received must be complete. Provide 'day' of the date."},
-            {'input': {'year': '1998', 'month': '03', 'day': '14', 'type': 'rev-request'},
-             'result': 'ok',
-             'element': 'rev-request',
-             'object_date': date(1998, 3, 14)},
-            {'input': {'year': '1998', 'month': '05', 'day': '24', 'type': 'rev-recd'},
-             'result': 'ok',
-             'element': 'rev-recd',
-             'object_date': date(1998, 5, 24)},
-            {'input': {'year': '1998', 'month': '06', 'day': '06', 'type': 'accepted'},
-             'result': 'ok',
-             'element': 'accepted',
-             'object_date': date(1998, 6, 6)},
-            {'input': {'year': '2012', 'month': '06', 'day': '01', 'type': 'approved'},
-             'result': 'ok',
-             'element': 'approved',
-             'object_date': date(2012, 6, 1)}
+            {
+                'title': 'History date validation',
+                'xpath': './/front//history//date',
+                'validation_type': 'value',
+                'response': 'ERROR',
+                'expected_value': ['received', 'rev-request', 'rev-recd', 'accepted', 'corrected'],
+                'got_value': ['rev-request', 'received', 'rev-recd', 'accepted', 'corrected'],
+                'message': "Got [('rev-request', '1998-01-04'), ('received', '1998-01-05'), ('rev-recd', '1998-05-24'),"
+                           " ('accepted', '1998-06-06'), ('corrected', '2012-06-01')]",
+                'advice': 'Provide ordering of events',
```

Comment:
@Rossi-Luciano corrige a mensagem: "Provide the dates according to the correct chronological order"

Reply:
OK, @robertatakenaka